Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/SearchTry.hs

```diff
@@ -82,6 +82,11 @@ initiateDriverSearchBatch merchant searchReq tripCategory vehicleVariant estOrQu
               driverSelectedFare = Nothing,
               customerExtraFee = customerExtraFee,
               nightShiftCharge = Nothing,
+              customerCancellationDues = customerCancellationDues,
+              nightShiftOverlapChecking = False, -- sending True in rental
+              estimatedDistance = searchReq.estimatedDistance,
+              estimatedRideDuration = searchReq.estimatedDuration,
+              timeDiffFromUtc = Nothing,
```

Comment:
Why it needed to be there? It could be a merchant wise config and use from there only.

Reply:
It is, I'm just passing from here as TC is not present in fareCalculator.